README.md

```diff
@@ -4,7 +4,42 @@
 Nanotube: data router for Graphite
 =================================
 
-This is the router (or relay, or reverse-proxy) for Graphite. It routes incoming records according to the specified rules. Nanotube is designed for high-load systems. It is used at Booking.com to route up to `400k` incoming and `1.2M` outgoing records/sec on a single box.
+This is the router (or relay, or reverse-proxy) for Graphite. It routes incoming records according to the specified rules. The Nanotube is designed for high-load systems. It is used at Booking.com to route up to `400k` incoming and `1.2M` outgoing records/sec on a single box.
+
+Build and run
+-------------
+
+1. Get this repository with
+`go get -u github.com/bookingcom/nanotube`
+2. Navigate to it and run
+`make`
+3. Run it with
+
+`./nanotube -config config/config.toml -clusters config/clusters.toml -rules config/rules.toml`
+
+Running with docker-compose
+-----------------------------------
+
+Run the input stack `nanotube` -> `go-carbon-*` and the query stack to check on those inputs `carbonapi` -> `zipper` -> `go-carbon` with:
```

Comment:
Oh, I think the second part of the sentence is not needed any more ("and the query stack ...")
I forgot to remove that 😅